REFERENCE.md

```diff
@@ -10,6 +10,7 @@
 * [`profile_system_auth::config`](#profile_system_authconfig): Configure system auth
 * [`profile_system_auth::kerberos`](#profile_system_authkerberos): Basic kerberos client setup, with optional host principal management
 * [`profile_system_auth::ldap`](#profile_system_authldap): Basic ldap client setup
+* [`profile_system_auth::su`](#profile_system_authsu): Basic su configuration
```

Comment:
This apparently didn't get added as part of a previous commit. It is unrelated to the current commit.